client/src/components/FeedbackButton.jsx

```diff
@@ -10,6 +10,7 @@ class FeedbackButton extends Component {
     );
   }
 
+  // TODO: Move setTimeout reset functionality to back-end
```

Comment:
Will either move to PresenterView or back-end.  Discussion completed with Ross, Ari, and Johnny